mt_rng.cpp

```diff
@@ -0,0 +1,53 @@
+#include <ctime> // Include for time() function
+
+using namespace std;
+
+typedef unsigned long long ulong;
+
+unsigned int MT[624]; // MT array
+int mti = 0; // index
+
+// Init MT seed
+void InitMT(unsigned int x0)
+{
+    ulong x;
+
+    MT[0] = x0;
+    for (int i = 1; i < 623; i++)
+    {
+        x = MT[i - 1];
+        x = (23023 * x) & 0xffffffffull;
+        x = (3 * x) & 0xffffffffull;
+        MT[i] = x;
+    }
+}
+
+// Rand MT
+void RandomizeMT()
```

Comment:
Do we need this function ? 
I think we would like to control the seed to recreate same results in other tests.

Reply:
Just one layer of abstraction. We can delete it.

Reply:
Yeah we can, also we can not do it, and just call the InitMT() function with time(NULL).